### DISCUSSION TITLE
Why expose the `softmax` and `sigmoid` parameters on the `DiceFocalLoss` if by default the `FocalLoss` works only with logits?

### DISCUSSION THREAD

**Discussion by ramonemiliani93** [Project-MONAI/MONAI#2612] [General]
Hi, I find it confusing that the `softmax` and `sigmoid` parameters are exposed on the `DiceFocalLoss` if by default it has to work with the logits, as per the documentation:
```
input: the shape should be BNH[WD]. The input should be the original logits
                due to the restriction of ``monai.losses.FocalLoss``.
```
Is there a specific reason for not removing the parameters? I feel this may lead to bugs on user code.

**Comment by Nic-Ma** [comment]
Hi @ramonemiliani93 ,

The `DiceFocalLoss` is a hybrid loss function with `DiceLoss` and `FocalLoss`, the activation parameter is used in the `DiceLoss` of `DiceFocalLoss`.
Let me try to enhance the doc-string to make it more clear later.

Thanks.

**Reply by Nic-Ma** [reply]
Submitted PR https://github.com/Project-MONAI/MONAI/pull/2614 for it.
Thanks.
